crates/orderbook/src/api/post_solver_competition.rs

```diff
@@ -8,18 +8,6 @@ use shared::api::convert_json_response_with_status;
 use std::{convert::Infallible, sync::Arc};
 use warp::{reply::with_status, Filter, Rejection};
 
-#[async_trait::async_trait]
-pub trait SolvableOrdersCache: Send + Sync {
-    async fn update_next_solver_competition_id(&self) -> anyhow::Result<()>;
-}
```

Comment:
This was added recently by me to fix a bug where the driver could run too quickly which causes it to not use the right auction id. This way of solving the problem no longer works because the orderbook cannot signal the autopilot. I will fix this again by making a change to the driver in a follow up PR. Not having the fix is not a serious issue it just means that sometimes the solver competition id as given to the solvers is off by one.